code/game/machinery/doors/door.dm

```diff
@@ -226,7 +249,29 @@
 /obj/machinery/door/proc/try_to_crowbar(mob/user, obj/item/I)
 	return
 
+/obj/machinery/door/proc/clean_cmag_ooze(obj/item/I, mob/user) //Emags are Engineering's problem, cmags are the janitor's problem
+	var/cleaning = FALSE
+	if(istype(I, /obj/item/reagent_containers/spray/cleaner))
+		var/obj/item/reagent_containers/spray/cleaner/C = I
+		if(C.reagents.total_volume >= C.amount_per_transfer_from_this)
+			cleaning = TRUE
+		else
+			return
+	if(istype(I, /obj/item/soap))
+		cleaning = TRUE
+
+	if(!cleaning)
+		return
+	user.visible_message("<span class='notice'>[user] starts to clean the ooze off the access panel.</span>", "<span class='notice'>You start to clean the ooze off the access panel.</span>")
+	if(do_after(user, 50, target = src))
+		user.visible_message("<span class='notice'>[user] cleans the ooze off [src].</span>", "<span class='notice'>You clean the ooze off [src].</span>")
+		cmagged = FALSE
+	return
+
 /obj/machinery/door/attackby(obj/item/I, mob/user, params)
+	if(cmagged)
+		clean_cmag_ooze(I, user)
```

Comment:
```suggestion
		clean_cmag_ooze(I, user)
		return
```
I believe you meant to do this? If not, just resolve and ignore this suggestion.

Reply:
Adding a return there would prevent this proc from checking for fireaxes on cmagged doors. I'll be adding a method to do `if(cmagged && I.canclean)` instead, which _would_ be able to return without issue, but that's out of scope of this PR.